### PR TITLE
Minor minor pr, change gravity too

### PR DESCRIPTION
**Pull request by peasant98**
This PR adds logging of the DH params and fixes a bug in data collection.

### CODE REVIEW

**Comment by watakandai** [comment]
You changed the optimizer in this PR right?

**Comment by peasant98** [comment]
I believe I already updated it in master. Don't review yet

**Comment by watakandai** [comment]
what was this PR for?

**Comment by peasant98** [comment]
Some bug fixes, mainly. 

Also `logging.info(f'Params: {params}')` is needed so we can get the params for our results.

File: config/panda.yaml [hunk 1]
```diff
@@ -18,25 +18,25 @@ su_dh_parameter:
...
   su6:
-    position: [-0.023, -0.030, 1.041]
-    rotation: [0.482, -0.482, -0.517, 0.517]
+    position: [0.039, -0.060, 1.037]
+    rotation: [0.521, -0.478, 0.481, 0.519]
   su7:
-    position: [0.165, 0.000, 1.028]
-    rotation: [0.732, 0.000, 0.682, -0.000]
+    position: [0.118, 0.047, 1.031]
+    rotation: [-0.707, 0.037, 0.020, 0.706]
```
Comment (watakandai) [inline config/panda.yaml]: What are all these changes?

Comment (peasant98) [reply config/panda.yaml]: from some different imu poses, I think. Probably not needed.

File: robotic_skin/calibration/optimizer.py [hunk 2]
```diff
@@ -383,7 +383,6 @@ def optimize(self, data):
 
             # optimize parameters wrt data
             params = self._optimize(i_su=i_su)
-
```
Comment (peasant98) [inline robotic_skin/calibration/optimizer.py]: might be a mistake here @peasant98

File: robotic_skin/calibration/optimizer.py [hunk 3]
```diff
@@ -554,7 +553,6 @@ def _optimize(self, i_su):
         local_opt = nlopt.opt(C.LOCAL_OPTIMIZER, self.n_param)
         opt.set_local_optimizer(local_opt)
         params[self.target_index] = opt.optimize(params[self.target_index])
-
```
Comment (peasant98) [inline robotic_skin/calibration/optimizer.py]: ditto

File: config/panda.yaml (hunk 1 above)
Comment (peasant98) [reply config/panda.yaml]: fixed

**Review by watakandai — APPROVED** [review]

**Pull request merged**